src/libexpr/primops.cc

```diff
@@ -4857,7 +4857,10 @@ void EvalState::createBaseEnv()
     addConstant("__nixPath", v, {
         .type = nList,
         .doc = R"(
-          The value of the [`nix-path` configuration setting](@docroot@/command-ref/conf-file.md#conf-nix-path): a list of search path entries used to resolve [lookup paths](@docroot@/language/constructs/lookup-path.md).
+          A list of search path entries used to resolve [lookup paths](@docroot@/language/constructs/lookup-path.md).
+          Its value is primarily determined by the [`nix-path` configuration setting](@docroot@/command-ref/conf-file.md#conf-nix-path), which can be
+          - Overridden by the [`NIX_PATH`](@docroot@/command-ref/env-common.md#env-NIX_PATH) environment variable or the `--nix-path` option
+          - Extended by the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) or `--extra-nix-path`
```

Comment:
```suggestion
          - Overridden using the [`NIX_PATH`](@docroot@/command-ref/env-common.md#env-NIX_PATH) environment variable or the `--nix-path` option
          - Extended using the [`-I` option](@docroot@/command-ref/opt-common.md#opt-I) or `--extra-nix-path`
```


Not sure if we want to assign agency to variables. Can+using makes more sense to me.

Reply:
We could also say "*are* overridden by" -- I don't have an issue with talking about software entities as if they were agents, this is a common simplification that is efficient if the context is clear.

Reply:
Committed the alternative.

I'll try to pick somewhat fewer nits btw.